OmniKit/Model/FaultEventCode.swift

```diff
@@ -311,7 +311,7 @@ public struct FaultEventCode: CustomStringConvertible, Equatable {
                 }
             }()
         } else {
-             return "Unknown Error Code"
+            faultDescription = "Fault Event Code %1$02x: Unknown Error Code"
```

Comment:
This is not needed... The code is prepended below.

Reply:
I see it, I am committing too fast, time to go to bed. :-)